src/OAuth/Tokens/PenneoTokenValidator.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Penneo\SDK\OAuth\Tokens;
+
+use Carbon\Carbon;
+
+class PenneoTokenValidator
+{
+    private const TOKEN_EXPIRY_BUFFER_IN_SECONDS = 5;
+
+    public static function isValid(?PenneoTokens $tokens = null): bool
```

Comment:
You are not checking if the token is valid but just if it is not expired. I also have a difficult time handling the fact that this class is called something with token in singularity but is asserts multiple tokens (plural)

Reply:
Grammatically, "token validator" is correct, while "tokens validator" is not (just like we don't say "books shelf" but instead we say "book shelf"), but this is the second comment I have received about this, so for the sake of clarity, I am renaming the class to `PenneoTokensValidator`.

I've also renamed the method `isValid` to `areNotExpired` to more precisely describe what it does.